### DISCUSSION TITLE
Center of Loco deck and center of Crazyflie?

### DISCUSSION THREAD

**Discussion by danteppc** [bitcraze/discussions#783] [Q&A] [answered]
Hi,

Does a position estimate in Crazyflie tell you the position of Loco deck (antenna maybe?) or the center of vehicle (the drone)?

From [here](https://www.bitcraze.io/documentation/system/platform/cf2-coordinate-system/), it seems the center of Crazyflie is what x,y,z show. But I wonder how it should be interpreted when the UWB sensor is few centimeters farther than the center of body, especially given that the Loco nodes have the UWB radio at the tip of its rectangular body. I understand that the difference may not be that significant but just to better understand the system. 

I hope I was clear enough.

Thanks!

**Comment by krichardsson** [comment, marked as answer]
Hi,
Yes, you are right, all measurements are based on the position of the antenna, and there is no compensation for the offset in the physical setup.

I doubt it makes any difference though on a Crazyflie as it is so close, but on a larger platform it could maybe be interesting to compensate if the Loco deck is mounted "far" away. Considering that the errors in measurements due to the orientation of antennas (CF to anchor) might be in the range of over 10-20 cm, the small offset can safely be ignored.
